src/wallet/rpc/encrypt.cpp

```diff
@@ -220,7 +220,11 @@ RPCHelpMan encryptwallet()
                 "After this, any calls that interact with private keys such as sending or signing \n"
                 "will require the passphrase to be set prior the making these calls.\n"
                 "Use the walletpassphrase call for this, and then walletlock call.\n"
-                "If the wallet is already encrypted, use the walletpassphrasechange call.\n",
+                "If the wallet is already encrypted, use the walletpassphrasechange call.\n"
+                "** IMPORTANT **\n"
+                "For security reasons, the encryption process will generate a new HD seed, leading to the creation of a fresh set of active descriptors. "
```

Comment:
```suggestion
                "For security reasons, the encryption process will generate a new HD seed, leading to the creation of a fresh set of active descriptors.\n"
```
nit, new line forgotten?
(sry didn't notice at previous review)